indexer/services/socks/src/config.ts

```diff
@@ -45,6 +45,11 @@ export const configSchema = {
   COMLINK_URL: parseString(),
   AXIOS_TIMEOUT_MS: parseInteger({ default: 5000 }), // 5 seconds
   INITIAL_GET_TIMEOUT_MS: parseInteger({ default: 20_000 }), // 20 seconds
+
+  // Restrictions
+  RESTRICTED_COUNTRIES: parseString({
```

Comment:
why can't socks just depend on comlink for the RESTRICTED_COUNTRIES to be blocked there?

Reply:
`socks` queries `comlink` for initial data payload, and the origination of that request wouldn't be from a geoblocked region.